Barlist/Report.h

```diff
@@ -32,7 +32,13 @@ class CReport
 
    static CString GetReportHeader();
 
-   void BuildReport(IBarlist* pBarlist);
+   enum ReportOptions {
+	   REPORT_TOTAL_QUANTITIES = 1,
+	   REPORT_GROUP_QUANTITIES = 2,
+	   REPORT_TOTAL_AND_GROUP_QUANTITIES = 3
+   };
+
+   void BuildReport(IBarlist* pBarlist, int reportOptions);
```

Comment:
Use enum type instead of int. With int, the call can pass in -500. If the type is the 'enun class' the caller is constrained to only the valid enum values.

Reply:
Ok. I've changed it to enum class CReport::ReportOptions